### PR TITLE
Add typing indicator test

### PR DESCRIPTION
**Pull request by betsyknoke**
Add Api function for connecting/disconnecting from a conversation.
Add Api function for answering a conversation.
Add Api function for sending a typing indicator.
Add Api function for getting and sending wrap up codes.
Add typing indicator test.

### CODE REVIEW

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/page/MessengerPage.kt [hunk 1]
```diff
@@ -95,27 +86,13 @@ class MessengerPage(activity: Activity) : BasePage(activity) {
...
-
     // Verify response for sending a message
     fun checkSendMsgFullResponse() {
         val response = getFullResponse()
         if (!(response.contains(messageResultText1, ignoreCase = true))) AssertionError("Response does not contain MessageUpdated")
         if (!(response.contains(messageResultText2, ignoreCase = true))) AssertionError("Response does not contain MessageUpdated")
     }
 
-    // Verify response for the ping command
-    fun checkPingFullResponse() {
```
[the remark this answers is not in the document]
Comment (betsyknoke) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/page/MessengerPage.kt:112]: I did add a healthCheck test. It is found on line 62 in ComposePrototypeUITest.kt and called in the testAllCommands test in line 161.

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/support/ApiHelper/API.kt [hunk 2]
```diff
@@ -25,7 +25,7 @@ class API : IdlingResource {
         with(url.openConnection() as HttpURLConnection) {
             requestMethod = httpMethod
             setRequestProperty("Authorization", "bearer $token")
-            setRequestProperty("User-Agent", "testing-communicate-mobile-Android")
+            setRequestProperty("User-Agent", "Android-MMSDK-Testing")
```
[the remark this answers is not in the document]
Comment (betsyknoke) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/support/ApiHelper/API.kt]: Updated

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/support/ApiHelper/Conversations.kt [hunk 3]
```diff
@@ -81,32 +78,68 @@ fun API.getConversationInfo(conversationId: String): Conversation {
     return parseJsonToClass(result)
 }
 
-fun API.sendConnect(conversationInfo: Conversation) {
-    val statusPayload = "{\"state\": \"CONNECTED\"}".toByteArray()
-    sendConversationStatus(payload = statusPayload, conversationInfo = conversationInfo)
+fun API.sendTypingIndicator(conversationInfo: Conversation) {
```
[the remark this answers is not in the document]
Comment (betsyknoke) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/support/ApiHelper/Conversations.kt]: Updated

File: androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt [hunk 4]
```diff
@@ -127,6 +112,30 @@ class ComposePrototypeUITest : BaseTests() {
...
+            enterDeploymentID()
+            selectView(testBedViewText)
+        }
+        messenger {
+            verifyPageIsVisible()
+        }
+        connect()
+        sendMsg("howdy")
+        val conversationInfo = apiHelper.answerNewConversation()
+        apiHelper.sendTypingIndicator(conversationInfo!!)
```
[the remark this answers is not in the document]
Comment (betsyknoke) [reply androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt]: Yes. Updated

**Review by AfanasievAnton — APPROVED** [review]

**Review by crumpf — APPROVED** [review]

**Review by mmorehou — APPROVED** [review]

**Pull request merged**